docker/Dockerfile-workers

```diff
@@ -16,6 +32,13 @@ RUN git clone https://github.com/operator-framework/operator-registry $OPM_SRC &
     (cd $OPM_SRC && git checkout $OPM_TAG && make build) && \
     cp $OPM_SRC/bin/opm /usr/bin/opm
 
+COPY ./docker/podman-1.9.3-2.module+el8.2.1+6867+366c07d6.src.rpm /tmp/podman-1.9.3-2.module+el8.2.1+6867+366c07d6.src.rpm
```

Comment:
Could you also add a comment above here with a link to the bug?

Reply:
done